### ISSUE TITLE
PDF & printed portfolio: Add case study for EBA

### ISSUE THREAD

**Issue by MangopearUK** [MangopearUK/Mangopear-Brochure-Theme#59]

**Comment by MangopearUK** [comment]
Update: **8th January 2020**

The case study still needs to be written. It's an low priority and is scheduled for the second release of the portfolio in 2020. Second release is aimed for later in January 2020.

**Comment by MangopearUK** [comment]
Update: **27th January 2020**

This case study has been completed for the printed/PDF portfolio, including every stage of the proof-reading. This case study now needs to be added to the website.

This can only happen after #64 has been completed though. Awaiting the completion of that task before we can proceed with the content population of this case study.
